README.md

```diff
@@ -1,4 +1,65 @@
-# data-pack
+# data-pack : data packages for Python
 
-Data packages for Python
+Data-pack is a library for querying and downloading data in Python.
 
+The data-pack specification is given below.  Packages can be
+distributed either directly (and queried via URL), or via an index.
+
+## Specification: data-pack
+
+A data pack is a JSON file, stored at a publicly accessible and
+*persistent* HTTP URL, of the following format:
+
+```
+{
+  "description": "Description of the package",
+  "version": 0.0,
+  "files": [
+    {
+      "name": "tipsy",
+      "url": "https://api.figshare.com/v2/articles/4810153",
```

Comment:
@carreau suggests allowing multiple URLs here, both mapping to the same file for possible redundancy